net/i2pd/Makefile

```diff
@@ -36,45 +37,49 @@ define Package/i2pd
 endef
 
 define Package/i2pd/description
-	I2P (Invisible Internet Protocol) is a universal anonymous network layer. All
-	communications over I2P are anonymous and end-to-end encrypted, participants
-	don't reveal their real IP addresses.
-endef
-
-define Package/i2pd/conffiles
-/etc/config/i2pd
-/etc/i2pd/i2pd.conf
-/etc/i2pd/tunnels.conf
-/etc/i2pd/tunnels.d/
-endef
-
-define Build/Prepare
-	$(Build/Prepare/Default)
-	# race condition
-	cd $(PKG_BUILD_DIR); $(MAKE) mk_obj_dir
+  I2P (Invisible Internet Protocol) is a universal anonymous network layer. All
+  communications over I2P are anonymous and end-to-end encrypted, participants
+  don't reveal their real IP addresses.
 endef
 
 TARGET_LDFLAGS+=-latomic
 
 define Package/i2pd/install
 	$(INSTALL_DIR) $(1)/usr/sbin
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/i2pd $(1)/usr/sbin
+
+	# init.d script
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) ./files/i2pd.init $(1)/etc/init.d/i2pd
+
+	# certificates
 	$(INSTALL_DIR) $(1)/usr/share/i2pd
 	$(CP) $(PKG_BUILD_DIR)/contrib/certificates  $(1)/usr/share/i2pd
+
+	# service configuration
 	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/i2pd.config $(1)/etc/config/i2pd
+	$(INSTALL_CONF) ./files/i2pd.config $(1)/etc/config/i2pd
+
+	# i2pd configuration
 	$(INSTALL_DIR) $(1)/etc/i2pd
-	$(INSTALL_CONF) $(PKG_BUILD_DIR)/contrib/i2pd.conf $(1)/etc/i2pd
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/contrib/i2pd.conf $(1)/etc/i2pd/i2pd.conf.example
 	$(SED) ' \
 		s/127.0.0.1/192.168.1.1/g; \
 		s/datadir = \/var\/lib/datadir = \/etc/ \
-	' $(1)/etc/i2pd/i2pd.conf
-	$(INSTALL_CONF) $(PKG_BUILD_DIR)/contrib/tunnels.conf $(1)/etc/i2pd
+	' $(1)/etc/i2pd/i2pd.conf.example
+	# optimal preconfigured config
+	$(INSTALL_DATA) ./files/i2pd.conf $(1)/etc/i2pd
+
+	# tunnels config
 	$(INSTALL_DIR) $(1)/etc/i2pd/tunnels.d
-	$(INSTALL_CONF) $(PKG_BUILD_DIR)/contrib/tunnels.d/*.conf  $(1)/etc/i2pd/tunnels.d
-	# subscriptions.txt deprecated, see #8088
-	$(INSTALL_DIR) $(1)/etc/init.d
-	$(INSTALL_BIN) ./files/i2pd.init $(1)/etc/init.d/i2pd
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/contrib/tunnels.conf $(1)/etc/i2pd
+endef
+
+define Package/i2pd/conffiles
```

Comment:
Why you moved it below?

Reply:
Because I've seen such behaviour in `net/bind`, `net/dnscrypt-proxy2`.

Reply:
Otherwise like ``admin/netdata``, ``admin/openwisp-config``, ``admin/syslog-ng`` has it above. It seems that sometimes it is above, sometimes it is below. Well, it could stay where it is. I don't have strong opinion on this one.

Reply:
Moved it above install section.